jobs/haproxy/templates/haproxy.config.erb

```diff
@@ -822,7 +822,12 @@ backend http-routed-backend-<%= prefix_hash %>
 
 # TCP Routing  {{{
 <% if_link("tcp_router") do |tcp_router| -%>
-
+  <%-
+  resolvers = ""
+  if_p("ha_proxy.resolvers") do
+    resolvers = "resolvers default "
```

Comment:
Please, remove the whitespace at the end of "resolvers default "
`resolvers = "resolvers default"`